www/_blog/2021-11-29-community-day.mdx

```diff
@@ -0,0 +1,118 @@
+
+The beauty of building in the open is that the community can get involved in adding new OAuth providers, which is exactly what [@TheHarryET](https://twitter.com/TheHarryET) and [MonsterDeveloper](https://github.com/MonsterDeveloper) have been doing. Thanks to them you can now provide sign-in with [Slack](https://supabase.com/docs/guides/auth/auth-slack) as well as [Spotify](https://supabase.com/docs/guides/auth/auth-spotify) to your users, as well as use [MessageBird](https://supabase.com/docs/guides/auth/auth-messagebird) for phone auth as an alternative to the existing [Twilio](https://supabase.com/docs/guides/auth/auth-twilio) integration.
+
+To celebrate this, we've created a [Supabase Spotify playlist](https://open.spotify.com/playlist/6FMx89EJjNHvjhy5auyZHx?si=e5ff6428415f420b) for y'all to blast out when reading our Launch Week posts, as well as when building with Supabase. Some top coding tunes in here! 📻
+
+<iframe
+  className="w-full"
+  width="700"
+  height="350"
+  src="https://open.spotify.com/embed/playlist/6FMx89EJjNHvjhy5auyZHx"
+  frameBorder="0"
+  allowtransparency="true"
+  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+## SupaSquad updates
+
+The [SupaSquad](https://supabase.com/docs/handbook/supasquad) is an official Supabase advocate program where community members help build and manage the Supabase community. 
+
+Among many other awesome things, the SupaSquad has been absolutely on fire building [client libraries](https://supabase.com/docs/reference/javascript/supabase-client) for all flavors of backends and environments.
+
+![client-libs.jpg](/images/blog/launch-week-three/community-day/client-libs.jpg)
+
+### Python client library updates
+
+- New maintainers [`@Dreinon`](https://github.com/dreinon), [`@anand2312`](https://github.com/anand2312) and [`@leynier`](https://github.com/leynier). 💚
+- Gotrue-py rewritten and now at feature parity with gotrue-js.
+- Postgrest-py now support synchronous operations as well instead of just synchronous operations in the past. Also at feature-parity with postgrest-js.
+- Supabase-py is now Supabase (e.g. you do `pip3 install supabase` instead of `pip3 install supabase-py`).
+- Storage is working for the py client library but it's yet to be extracted out as a standalone lib.
+
+## New tutorials and integration guides
+
+Our own [Jon Meyers](https://twitter.com/_dijonmusters) and the broader community have been busy creating awesome tutorials and integration guides to help everyone build even more amazing things with Supabase. Jon has added official guides for working with [Auth0](https://supabase.com/docs/guides/integrations/auth0), [Draftbit](https://supabase.com/docs/guides/integrations/draftbit), [Prisma](https://supabase.com/docs/guides/integrations/prisma), and [Vercel](https://supabase.com/docs/guides/integrations/vercel)!
```

Comment:
Just a small one, I think the Prisma guide was @MildTomato, and the Draftbit one was from @amandeepmittal. Not sure if he is from draftbit or just the Supabase community.

Maybe change to:

Our own [Jon Meyers](https://twitter.com/_dijonmusters) and the broader community have been busy creating awesome tutorials and integration guides to help everyone build even more amazing things with Supabase. Jon has added official guides for working with [Auth0](https://supabase.com/docs/guides/integrations/auth0) and [Vercel](https://supabase.com/docs/guides/integrations/vercel), [Jonny Summers-Muir](https://twitter.com/JSummersMuir) has been busy paving the way for [Prisma](https://supabase.com/docs/guides/integrations/prisma), and [Aman Mittal](https://github.com/amandeepmittal) has hooked us up with a guide for [Draftbit](https://supabase.com/docs/guides/integrations/draftbit)!